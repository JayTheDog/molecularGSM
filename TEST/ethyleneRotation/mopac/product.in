PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  C 0.664985 0.008774 0.002231
  C -0.665029 -0.009222 -0.002232
  H -1.233934 -0.057311 -0.926987
  H 1.242716 -0.040253 -0.916864
  H 1.233719 0.069972 0.926154
  H -1.242436 0.031922 0.917487
