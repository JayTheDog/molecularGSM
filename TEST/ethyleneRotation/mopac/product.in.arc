

                     SUMMARY OF PM6 CALCULATION, Site No: 14762

                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:52:11 2016
                                                       No. of days left =   0

           Empirical Formula: C2 H4  =     6 atoms

PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =         15.70754 KCAL/MOL =      65.72034 KJ/MOL
          TOTAL ENERGY            =       -298.56828 EV
          ELECTRONIC ENERGY       =       -747.56452 EV
          CORE-CORE REPULSION     =        448.99625 EV
          GRADIENT NORM           =          0.26554
          DIPOLE                  =          0.00118 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =          6
          IONIZATION POTENTIAL    =         10.675164 EV
          HOMO LUMO ENERGIES (EV) =        -10.675  1.158
          MOLECULAR WEIGHT        =         28.054
          COSMO AREA              =         71.33 SQUARE ANGSTROMS
          COSMO VOLUME            =         53.05 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     5    H     3     3.10034
            H     6    H     4     2.93420
            H     5    H     4     0.00199
          SCF CALCULATIONS        =         11
          WALL-CLOCK TIME         =          0.031 SECONDS
          COMPUTATION TIME        =          0.034 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  C     0.66357486 +1   0.00817357 +1   0.00256142 +1
  C    -0.66353768 +1  -0.00741427 +1  -0.00261430 +1
  H    -1.25665257 +1  -0.06286327 +1  -0.90551088 +1
  H     1.26428598 +1  -0.03452061 +1  -0.89597058 +1
  H     1.25658790 +1   0.06493759 +1   0.90540321 +1
  H    -1.26443656 +1   0.03260861 +1   0.89604732 +1
 
