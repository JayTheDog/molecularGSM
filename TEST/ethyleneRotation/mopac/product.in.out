 *******************************************************************************
 ** Site#: 14762        For non-commercial use only     Version 15.168L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.168L web: HTTP://OpenMOPAC.net         Days remaining:   0    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Dec  2 19:52:11 2016  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  NOSYM      - POINT-GROUP SYMMETRY SET TO C1
 *  T=         - A TIME OF     10.0 MINUTES REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.66498500  *   0.00877400  *   0.00223100  *
     2       C         -0.66502900  *  -0.00922200  *  -0.00223200  *
     3       H         -1.23393400  *  -0.05731100  *  -0.92698700  *
     4       H          1.24271600  *  -0.04025300  *  -0.91686400  *
     5       H          1.23371900  *   0.06997200  *   0.92615400  *
     6       H         -1.24243600  *   0.03192200  *   0.91748700  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        0.6650    0.0088    0.0022
     2         C       -0.6650   -0.0092   -0.0022
     3         H       -1.2339   -0.0573   -0.9270
     4         H        1.2427   -0.0403   -0.9169
     5         H        1.2337    0.0700    0.9262
     6         H       -1.2424    0.0319    0.9175

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    6

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    12.722 HEAT:  16.03404
 CYCLE:     2 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     9.368 HEAT:  15.87789
 CYCLE:     3 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     9.428 HEAT:  15.81772
 CYCLE:     4 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     7.299 HEAT:  15.72915
 CYCLE:     5 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     4.615 HEAT:  15.71502
 CYCLE:     6 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     3.189 HEAT:  15.70930
 CYCLE:     7 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     0.266 HEAT:  15.70754

     GRADIENT =  0.26554  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:52:11 2016
                                                       No. of days left =   0




          FINAL HEAT OF FORMATION =         15.70754 KCAL/MOL =      65.72034 KJ/MOL


          TOTAL ENERGY            =       -298.56828 EV
          ELECTRONIC ENERGY       =       -747.56452 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =        448.99625 EV
          COSMO AREA              =         71.33 SQUARE ANGSTROMS
          COSMO VOLUME            =         53.05 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.26554
          IONIZATION POTENTIAL    =         10.675164 EV
          HOMO LUMO ENERGIES (EV) =        -10.675  1.158
          NO. OF FILLED LEVELS    =          6
          MOLECULAR WEIGHT        =         28.054

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     5    H     3     3.10034
            H     6    H     4     2.93420
            H     5    H     4     0.00199


          SCF CALCULATIONS        =         11
 
          WALL-CLOCK TIME         =          0.031 SECONDS
          COMPUTATION TIME        =          0.034 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C          0.66357486  *   0.00817357  *   0.00256142  *
     2       C         -0.66353768  *  -0.00741427  *  -0.00261430  *
     3       H         -1.25665257  *  -0.06286327  *  -0.90551088  *
     4       H          1.26428598  *  -0.03452061  *  -0.89597058  *
     5       H          1.25658790  *   0.06493759  *   0.90540321  *
     6       H         -1.26443656  *   0.03260861  *   0.89604732  *

                             CARTESIAN COORDINATES

   1    C        0.663574863     0.008173567     0.002561424
   2    C       -0.663537681    -0.007414272    -0.002614296
   3    H       -1.256652570    -0.062863274    -0.905510876
   4    H        1.264285985    -0.034520610    -0.895970582
   5    H        1.256587897     0.064937594     0.905403206
   6    H       -1.264436560     0.032608612     0.896047319


           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -26.86339 -20.59807 -16.01630 -14.46327 -12.04379 -10.67516   1.15849   5.46036
   5.74038   6.30295   6.49196   6.50631


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.286413        4.2864     1.11644     3.16997
    2          C          -0.286442        4.2864     1.11646     3.16999
    3          H           0.143223        0.8568     0.85678
    4          H           0.143198        0.8568     0.85680
    5          H           0.143227        0.8568     0.85677
    6          H           0.143207        0.8568     0.85679
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -0.000    -0.001     0.000     0.001
 HYBRID        -0.000    -0.000    -0.000     0.000
 SUM           -0.000    -0.001     0.000     0.001


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.11644   1.03207   1.00041   1.13750
    2  C   1.11646   1.03207   1.00041   1.13751
    3  H   0.85678
    4  H   0.85680
    5  H   0.85677
    6  H   0.85679

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.03 SECONDS

 == MOPAC DONE ==
